tests/testthat/test-quit_if_no_pacta_relevant_data.R

```diff
@@ -0,0 +1,51 @@
+test_that("`quit_if_no_pacta_relevant_data()` quits if no ABCD in non-interactive", {
+  skip_if_R_CMD_check()
+
+  tmp_dir <- withr::local_tempdir()
+  port_name <- "TestPort"
+  expect_null({
```

Comment:
Does this `expect_null()` call actually do anything? Can you just run `callr::r`, and then the `expect_true()` later?